the-simple-graphql-service/python/lambda_fns/loyalty.js

```diff
@@ -1,13 +1,12 @@
 "use strict";
 const { Lambda } = require('aws-sdk');
 exports.handler = async function (event) {
-    console.log("request:", JSON.stringify(event, undefined, 2));
-
-    var loyaltylevel = "Silver"
+    console.log("get loyalty request:", JSON.stringify(event, undefined, 2));
+    let loyaltylevel = "Silver";
     // return response back to upstream caller
     return sendRes(200, loyaltylevel);
 };
-const sendRes = (status, body) => {
+const sendRes = (status, loyaltylevel) => {
```

Comment:
This file does not appear to have been copied from the .js file compiled by the typescript folder. The parameter body is wrong and the function returned an error in the Python version.